dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sfdr/SfdrData.kt

```diff
@@ -1,19 +1,22 @@
 package org.dataland.datalandbackend.model.sfdr
 
+import com.fasterxml.jackson.annotation.JsonProperty
 import org.dataland.datalandbackend.annotations.DataType
-import org.dataland.datalandbackend.model.CompanyReport
 import org.dataland.datalandbackend.model.sfdr.categories.environmental.SfdrEnvironmental
+import org.dataland.datalandbackend.model.sfdr.categories.general.SfdrGeneral
 import org.dataland.datalandbackend.model.sfdr.categories.social.SfdrSocial
 
 /**
  * --- API model ---
- * Data class containing all fields of the questionnaire for the SFDR framework
+ * Fields of the sfdr framework.
  */
+
 @DataType("sfdr")
 data class SfdrData(
-    val social: SfdrSocial?,
+    @JsonProperty(value = "general", required = true)
```

Comment:
I have the feeling that the part "value = general" of this annotation is not necessary here.
The field will be parsed with the field-name of this data class by default.
Therefore it will be "general" by default. So why explicitly tell it again that it is "general" ?

Reply:
It is necessary so that incoming data can be parsed

Reply:
Clarified in a call with @stewal19 :   Somehow it does not work without the explicit "value = general" value.  We don't know why, but it is what it is.
Therefore I am closing this.